base/errorshow.jl

```diff
@@ -562,9 +562,9 @@ function replaceuserpath(str)
     return str
 end
 
-const STACKTRACE_MODULECOLORS = [:light_blue, :light_yellow,
-        :light_magenta, :light_green, :light_cyan, :light_red,
-        :blue, :yellow, :magenta, :green, :cyan, :red]
+const STACKTRACE_MODULECOLORS = [:magenta, :blue, :cyan, :green, :yellow]
+const STACKTRACE_FIXEDCOLORS = IdDict(Base => :red, Core => :red)
```

Comment:
I would avoid the use of `red` because it is typically used for errors.

Also, the light versions usually show better in general than the non-light versions.

Reply:
Re light versions, my concern is not to out-shine the function names too much, this is secondary information (which 1.5 doesn't print at all). I tried lots of terminal colour schemes, on dark ones usually `light_black` and `blue` are the most likely to be invisible. (Skipping `:blue` might be a good idea?) On light ones, `light_yellow`, `light_green` and `light_cyan` are the worst.

Error messages use `light_red`, and the thought was to echo that, but ideally not match. It seems nice to me to special-case Base/Core, partly because so often the first entry being `error` is different from the first entry being a package of interest. I tried a few options for this, printing no colour looks like a missing tooth...

<img width="1227" alt="Screenshot 2020-10-01 at 14 28 49" src="https://user-images.githubusercontent.com/32575566/94809135-72ec7480-03f2-11eb-9942-0f1f3025a319.png">

(PR above, master below, example from gist linked above.)

<img width="1225" alt="Screenshot 2020-10-01 at 14 34 20" src="https://user-images.githubusercontent.com/32575566/94809689-42f1a100-03f3-11eb-9793-225379f3cf28.png">

Reply:
The error messages no longer being printed in red is due to  https://github.com/JuliaLang/julia/pull/36024.